### ISSUE TITLE
Fails to create EXT4 FS!

### ISSUE THREAD

**Issue by notorand-it** [meefik/linuxdeploy#891]
I am trying to install ArchLinux AARM64 on a 4GB disk image.
The process fails because of some incompatibility with mke2fs:

```
+ /system/xbin/mke2fs -qF -O ^has_journal -t ext4 /sdcard/Linux/arch.img
/system/xbin/mke2fs: invalid option -- t
BusyBox v1.27.2-meefik (2017-08-24 10:35:24 MSK) multi-call binary.

Usage: mke2fs [-Fn] [-b BLK_SIZE] [-i INODE_RATIO] [-I INODE_SIZE] [-m RESERVED_PERCENT] [-L LABEL] BLOCKDEV [KBYTES]

	-b BLK_SIZE	Block size, bytes
	-F		Force
	-i RATIO	Max number of files is filesystem_size / RATIO
	-I BYTES	Inode size (min 128)
	-L LBL		Volume label
	-m PERCENT	Percent of blocks to reserve for admin
	-n		Dry run
+ is_ok fail done
```

I'll also try to attach the complete log file.

**Comment by notorand-it** [comment]
[log.txt](https://github.com/meefik/linuxdeploy/files/1583322/log.txt)

BusyBox is v1.27.2-meefik.
LinuxDeploy v2.0.4-228

**Comment by notorand-it** [comment]
On a 2gb image it works. I suspect an underlying fs issue (fat?).

**Comment by TCerbz** [comment]
In `FAT` or `FAT32` limit of maximum file size 4GB, try you use 3900 MB

**Comment by TCerbz** [comment]
In `FAT` or `FAT32` the maximum file size limit is 4GB, try using 3900 MB

**Comment by meefik** [comment]
Fixed in version 2.0.5

**Comment by notorand-it** [comment]
The limit on FAT is 2GB, not 4.
So you cannot create any disk image larger than 2GB until you convert you storage file system to EXT*.
